.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+  - "3.6"
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
```

Comment:
Will update this once we get our `setup.py` file

https://stackoverflow.com/questions/1471994/what-is-setup-py